common/web/input-processor/src/text/inputProcessor.ts

```diff
@@ -10,7 +10,11 @@ namespace com.keyman.text {
       baseLayout: 'us'
     }
 
-    private device: utils.DeviceSpec;
+    /**
+     * Indicates the device (platform) to be used for context-only rules,
+     * such as those found in `postkeystroke` and `newcontext` groups.
+     */
```

Comment:
Yeah, 'context-only' + 'device' => 'contextDevice' is exactly where my mind went and is kinda blocking me from thinking of a better name.

Technically, it _could_ be `displayDevice` (the device mode used for OSK display), or it could be `trueDevice` to match a similar property managed within the Web engine's DOM-based components.  Though it might be wise to keep it disentangled from the latter.

Reply:
Tricky. I think with the updated comment it's clear enough for now.